projects/packages/my-jetpack/_inc/data/use-jetpack-api-query.ts

```diff
@@ -0,0 +1,28 @@
+import restApi from '@automattic/jetpack-api';
+import { useQuery } from '@tanstack/react-query';
+import { useFetchingErrorNotice } from './notices/use-fetching-error-notice';
+
+const useJetpackApiQuery = < T >(
```

Comment:
Even though this uses TypeScript and the hook is quite short, I'd recommend adding a docblock for clarity